src/box/xrow.c

```diff
@@ -186,6 +186,9 @@ xrow_header_decode(struct xrow_header *header, const char **pos,
 			flags = mp_decode_uint(pos);
 			header->flags = flags;
 			break;
+		case IPROTO_STREAM_ID:
+			header->stream_id = mp_decode_uint(pos);
+			break;
```

Comment:
Can you please add a unit test? See test/unit/xrow.cc.

Reply:
I look in this test and have a question. Now i not encode stream_id in header and save it on the disk (see iproto_header_encode). Stream_id doesn't have any sense in the log as i think. Should i encode it and save it on the disk?

Reply:
You should not. It has nothing to do with persistency.

Reply:
Fixed